packages/headless/src/app/search-engine/jwt-reducer.ts

```diff
@@ -32,10 +33,6 @@ const possiblyWarnOnMismatch = (
     return;
   }
 
-  if (stateProp === tokenValue) {
-    return;
-  }
```

Comment:
Since the state prop will usually already be updated with the token's value, this would prevent us from warning the user that they are trying to set a different pipeline/searchhub than the one configured in the token.